src/include/function/arithmetic/arithmetic_functions.h

```diff
@@ -316,5 +317,11 @@ struct Pi {
     static inline void operation(double_t& result) { result = M_PI; }
 };
 
+struct Uuid {
```

Comment:
UUID is a short hand for `Universal Unique Identifier`, so either write all characters in upper case or in lower case.